File: app/src/main/java/tr/name/fatihdogan/books/api/response/ReadingPosition.java
package tr.name.fatihdogan.books.api.response;

import android.support.annotation.Keep;

import java.util.Date;

@Keep
@SuppressWarnings({"WeakerAccess", "unused"})
public class ReadingPosition {

    public static final String KIND_BOOKS_READINGPOSITION = "books#readingPosition";

    /**
     * {@link #KIND_BOOKS_READINGPOSITION}
     */
    public String kind;
    public String volumeId;
    public String gbTextPosition;
    public String gbImagePosition;
    public String epubCfiPosition;
    public String pdfPosition;
    public Date updated;

}

File: app/src/main/java/com/fthdgn/books/api/response/ReadingPosition.kt
@file:Suppress("unused")

package com.fthdgn.books.api.response

import android.support.annotation.Keep
import java.util.*

@Keep
class ReadingPosition {

    /**
     * [.KIND_BOOKS_READINGPOSITION]
     */
    var kind: String? = null
    var volumeId: String? = null
    var gbTextPosition: String? = null
    var gbImagePosition: String? = null
    var epubCfiPosition: String? = null
    var pdfPosition: String? = null
    var updated: Date? = null

    companion object {

        val KIND_BOOKS_READINGPOSITION = "books#readingPosition"
    }

}
